### PR TITLE
Profile sapling prototype

### PR DESCRIPTION
**Pull request by isabeltomb**
Update the primary color in the style sheet to match the color used
in the splinter logo.

Create a prototype of the profile sapling based on new designs. This
includes an additional page for the NewKey form. The profile sapling
does not function properly in this state and is intended only to be a
visual representation of the new sapling based on designs.

## **Testing:** 

Run splinter-ui in docker
`docker-compose -f docker-compose-biome.yaml up --build`

in a browser go to `http://localhost:3030` and click the `profile` tab.

### CODE REVIEW

File: saplings/profile/src/components/KeyTable.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+          </div>
+          <Link className="add-key" to="/profile/new-key">
+            <div className="icon"><Icon>add_icon</Icon></div>
+            New Key
+          </Link>
+        </div>
+        <div className="table-wrapper">
+        <table>
+          <tr>
```
Comment (peterschwarz) [inline saplings/profile/src/components/KeyTable.js]: This should be wrapped in `<thead>` (A console error occurs due to this)

File: saplings/profile/src/components/KeyTable.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <th>Key Name</th>
+            <th>Key</th>
+            <th>Created</th>
+            <th>Updated</th>
+            <th id="status-header">Status</th>
+            <th>Action</th>
+          </tr>
+          <tr>
```
Comment (peterschwarz) [inline saplings/profile/src/components/KeyTable.js]: The remaining rows should be wrapped in `<tbody>` (due to same warning)

**Review by peterschwarz — CHANGES_REQUESTED** [review]

File: saplings/profile/src/components/KeyTable.js (hunk 1 above)
Comment (isabeltomb) [reply saplings/profile/src/components/KeyTable.js]: fixed

File: saplings/profile/src/components/KeyTable.js (hunk 2 above)
Comment (isabeltomb) [reply saplings/profile/src/components/KeyTable.js]: fixed

**Review by peterschwarz — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**